src/blueprints/tv2/value-objects/tv2-action-manifest-data.ts

```diff
@@ -59,7 +60,7 @@ export interface Tv2VideoClipManifestData {
   fileName: string // userData.partDefinition.videoId
   durationFromIngest: number // userData.duration
   adLibPix: boolean // userData.adLibPix // What does "adLibPix" mean?
-  isVoiceOver: boolean // userData.voLevels
+  audioMode: Tv2AudioMode // userData.voLevels
```

Comment:
Should the comment still be there?

Reply:
You could argue yes and no. Yes, it still correspond to the `voLevels` of the `userData` (albeit it's a different type now). No, but then we should remove all the commets (which might be a reasonable thing to do).